v22.1/security-reference/authentication.md

```diff
@@ -57,12 +58,14 @@ Each rule definition contains up to 6 values.
 1. **`ADDRESS`** specifies the IP range which the rule will allow or block, either with the keyword "all", or with a valid IP address. The IP address can include an IP mask (the value of the field can be of the format XXX.XXX.XXX.XXX/X), or not, in which case the *next* value must be the mask (the value of this field will be of the form XXX.XXX.XXX.XXX, in which case the next field must be a valid IP mask).
 1. **`IP MASK`** (unless the Address in the prior field included or did not require an IP mask).
 1. Authentication **METHOD** by which specified user(s) may authenticate from specified addresses.
-  - `password`
-  - `cert`
-  - `cert-password`
-  - `gss`
-  - `reject`
-  - `trust`
+  - `password`: user may authenticate with a plain-text password.
```

Comment:
Here "plain-text" is hyphenated seems like elsewhere you've closed up, which I like